metrics/web/test/common/presentation/metrics_theme/state/theme_notifier_test.dart

```diff
@@ -13,11 +39,28 @@ void main() {
     });
 
     test(".changeTheme() notifies listeners about theme change", () {
-      final themeNotifier = ThemeNotifier();
+      final themeNotifier = ThemeNotifier(isDark: true);
 
       themeNotifier.addListener(expectAsync0(() {}));
 
       themeNotifier.changeTheme();
     });
+
+    test(
+        ".setTheme() changes the theme according to the given is dark value and notifies listeners",
```

Comment:
Seems like we should divide this test case into two separate as for the `changeTheme` method.